source/data-formats/time-series.txt

```diff
@@ -0,0 +1,200 @@
+.. _pymongo-time-series:
+
+================
+Time Series Data
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to store
+and interact with **time series data**.
+
+Time series data is composed of the following components:
+
+- Measured quantity
+- Timestamp for the measurement
+- Metadata that describes the measurement
+
+The following table describes sample situations for which you could store time
+series data:
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Situation
+     - Measured Quantity
+     - Metadata
+
+   * - Recording monthly sales by industry
+     - Revenue in USD
+     - Company, country
+
+   * - Tracking weather changes
+     - Precipitation level
+     - Location, sensor type
+
+   * - Recording fluctuations in housing prices
+     - Monthly rent price
+     - Location, currency
+
+.. _pymongo-time-series-create:
+
+Create a Time Series Collection
+-------------------------------
+
+.. important:: Server Version for Time Series Collections
+
+   To create and interact with time series collections, you must be
+   connected to a deployment running {+mdb-server+} 5.0 or later.
+
+You can create a time series collection to store time series data. To create
+a time series collection, pass the following parameters to the ``create_collection()``
+method:
+
+- Name of the new collection to create
+- ``timeseries`` argument 
+
+The ``timeseries`` argument is of type ``dict``. It contains the following fields:
+
+- ``timeField``
+- ``metaField``
+- ``granularity``
+- ``bucketMaxSpanSeconds``
+- ``bucketRoundingSeconds``
+
+See :manual:`Command Fields </reference/command/create/#command-fields>`
+to learn more about these fields.
+
+The following example shows how to create a time series collection using the 
+{+driver-short+}.
+
+Example
+~~~~~~~
+
+This example creates a time series collection named ``october2024`` with the
+``timeField`` option set to ``"timestamp"``.
```

Comment:
S: We typically use "the following" instead of "this" and introduce examples with a colon
```suggestion
The following example creates a time series collection named ``october2024`` with the
``timeField`` option set to ``"timestamp"``:
```